web/docs/tql2/operators/load_azure_blob_storage.md

```diff
@@ -0,0 +1,54 @@
+# load_azure_blob_storage
+
+Saves bytes to an Azure Blob Storage
```

Comment:
```suggestion
Loads bytes from an Azure Blob Storage.
```
Also, is it really "an", or rather "the"?

Reply:
There can be more than one Blob Store, and you are just saving to one of them.

Reply:
But it's "Azure Blob Storage", not "Azure Blob Store" (which doesn't give any results when searching for it).